app/src/screens/search-flow/AutoPathsForTeam.tsx

```diff
@@ -54,7 +54,7 @@ export const AutoPathsForTeam = ({route}) => {
           {autoPaths ? `Path ${currentIndex + 1} of ${autoPaths.length}` : ''}
         </Text>
         {autoPaths ? (
-          <CrescendoAutoViewer autoPath={autoPaths[currentIndex]} />
+          <ReefscapeViewer autoPath={autoPaths[currentIndex]} />
```

Comment:
what if someone tries to view a scout report from last year? add a field to the json in the auto path data column of supabase that defines which year the report was made in and conditionally display the appropriate viewer